packages/web3-utils/src/converters.ts

```diff
@@ -1,4 +1,4 @@
-import { validator, isAddress, isHexStrict } from 'web3-validator';
+import { validator, isAddress, isHexStrict, utils as validatorUtils } from 'web3-validator';
```

Comment:
Any benefit to doing `utils as validatorUtils` rather than just importing the specific methods used? If I understand correctly, without tree shaking, there is a lot of unused code being imported